orbit/pkg/packaging/windows_templates.go

```diff
@@ -57,6 +57,8 @@ var windowsWixTemplate = template.Must(template.New("").Option("missingkey=error
     <Property Id="FLEET_URL" Value="{{ if .FleetURL }}{{ .FleetURL }}{{ end }}"/>
     <Property Id="FLEET_SECRET" Value="dummy"/>
     <Property Id="ENABLE_SCRIPTS" Value="{{ if .EnableScripts }}True{{ else }}False{{ end }}"/>
+	<Property Id="FLEET_DESKTOP" Value="{{ if .Desktop }}True{{ else }}False{{ end }}"/>
+	<Property Id="END_USER_EMAIL" Value="{{ if .EndUserEmail }}{{ .EndUserEmail }}{{ else }}dummy{{end}}"/>
```

Comment:
I had to populate the property with something (`dummy`); otherwise, Wix would give errors.